core/src/main/resources/org/apache/spark/ui/static/historypage.js

```diff
@@ -78,6 +78,12 @@ jQuery.extend( jQuery.fn.dataTableExt.oSort, {
     }
 } );
 
+jQuery.extend( jQuery.fn.dataTableExt.ofnSearch, {
+    "appid-numeric": function ( a ) {
+        return a.replace(/[\r\n]/g, " ").replace(/<.*?>/g, "");
```

Comment:
Sorry for the confusion, yes I was suggesting what I think is a better approach using jQuery DataTables confs